### DISCUSSION TITLE
V 8.2.0.23 for MacOS Firefox is great but has some kinks

### DISCUSSION THREAD

**Discussion by tomdfrog** [aclap-dev/video-downloadhelper#202] [Help and Q&A]
### Your question

Did some searches and did not find mention of this.  Just sharing in case no one is aware.  The new version is otherwise working really well.  I mostly download from YT and the downloads are SUPER fast.  My VDH is licensed.  I have the CoApp installed and the preferences are set to Default.  MacOS Ventura.  Firefox 123

CONVERT:  The conversion tool will not run.  I click on the circle of Arrows, choose my files, select re-encode as h264, and when I click Convert, the Conversion window disappears.  It used to bring up a window asking for the destination for the re-encoded files.

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 4fff93d45bc34402
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/tfrogh_1",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by GrampaWildWilly** [comment]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3155, Firefox 123.0 64-bit, licensed VDH 8.2.1.3a1 beta, CoApp 2.0.19.  And I don't work for VDH.  I'm just another user.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/17d66f75-0e1e-4fb4-b314-41bf00c69a81)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/dc42a573-fbe9-4495-8fcf-0aef73da9490)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/effbe488-1d77-4522-81d4-b868e091395a)

It played fine in VLC.  You should post some URLs for which the mkv variants don't work for you.

**Comment by tomdfrog** [comment]
Well, it seems the  version 123.0 of Firefox which just installed... fixed the problem of 1440 and 2160 downloads.  Tested a couple of YT videos and they downloaded just fine as their 4k variant in .mkv

The Convert process is still not working.  After clicking Convert, the screen disappears instead of asking for a destination.

**Reply by paulrouget** [reply]
> The Convert process is still not working. After clicking Convert, the screen disappears instead of asking for a destination.

Is it the "Save As" button that's not working?

**Comment by tomdfrog** [comment]
There is no Save As button.  Only Convert.   I select my files...choose reencode...click Convert...window disappears.
![Screenshot 2024-02-28 at 5 19 14 PM](https://github.com/aclap-dev/video-downloadhelper/assets/160878838/1ac9feea-4009-4ff3-8cfe-426d855efedd)

**Reply by paulrouget** [reply]
Oh, for some reasons I assumed Download & Convert. I'll look into it.

**Reply by paulrouget** [reply]
@tomdfrog thanks for the useful feedback. I think I fixed the convert issue. See https://github.com/aclap-dev/video-downloadhelper/discussions/204#discussioncomment-8627340

I'm gonna close this issue and continue the conversation in #204.
